src/GPL-1.0.xml

```diff
@@ -0,0 +1,242 @@
+<spdx name="GNU General Public License v1.0 only" identifier="GPL-1.0" osi-approved="false">
+  <urls>
+    <url>http://www.gnu.org/licenses/old-licenses/gpl-1.0-standalone.html</url>
+  </urls>
+  <notes>This license was released: February 1989. This license has been deprecated. This refers to when this GPL 1.0
```

Comment:
Why is GPL-1.0 deprecated?  It [isn't deprecated in license-list][1] (although [GPL-1.0+ is deprecated][2]).

[1]: https://github.com/spdx/license-list/blob/788e54b566dc78358e3e1c3f9b7187ad4005114e/GPL-1.0.txt
[2]: https://github.com/spdx/license-list/blob/788e54b566dc78358e3e1c3f9b7187ad4005114e/deprecated_GPL-1.0%2B.txt